app/models/pusher.rb

```diff
@@ -126,6 +126,7 @@ def after_write
     Indexer.perform_later
     rubygem.delay.reindex
     GemCachePurger.call(rubygem.name)
+    StoreVersionContentsJob.perform_later(version) if ENV["STORE_VERSION_CONTENTS"] == "true"
```

Comment:
Only problem would be if adding that many jobs would burden the queue. It would be good to have this live and run a few manually before every gem push triggers a job. At the very least we want to make sure they are writing to the correct bucket and not doing anything dastardly before we run a few hundred or thousand jobs.

Reply:
Now that LaunchDarkly support is merged, maybe you can use a feature flag to control whether this runs or not?

Reply:
Added